label_studio/tests/e2e_actions.py

```diff
@@ -0,0 +1,159 @@
+    filename = case_config['filename']
+    headers = {
+        'Content-Type': 'multipart/form-data',
+    }
+    filepath = os.path.join(filepath, filename)
+    with open(filepath, 'rb') as file:
+        data = {
+            filename: (file, filename),
+        }
+
+        response = test_client.post('/api/import', data=data)
+    assert response.status_code == 201
+
+
+def action_import_test(test_client, case_config):
+    """
+        test
+        make sure impoted file is in project directory
+        and tasks r created
+    """
+    project = goc_project()
+    with open(project.config.get('input_path', None), 'r') as file:
+        data = file.read()
+
+
+def action_get_all_tasks(test_client, case_config):
+    """
+        action
+        get all tasks
+    """
+    #TODO get tasks
+    response = test_client.get('/api/projects/1/task_ids/')
+    data = json.loads(response.data.decode('utf-8'))
+    assert isinstance(data, list) == True
+
+
+def action_get_task(test_client, case_config):
+    """
+        action
+        get task by task_id
+    """
+    # get task by task_id
+    #TODO task_id to case_config
+    task_id = 0
+    response = test_client.get('/api/tasks/{task_id}/'.format(task_id=task_id))
+    assert response.status_code == 200
+
+
+def action_label(test_client, case_config):
+    """
+        action
+        send labeled data request
+    """
+    label_data = case_config['label_data']
+    #get task_id
+    response = test_client.get('/api/projects/1/task_ids/')
+    data = json.loads(response.data.decode('utf-8'))
+    task_id = data[-1]
+    response = test_client.get('/?task_id={task_id}'.format(task_id=task_id))
+    assert response.status_code == 200
+
+    headers = {
+        'Content-Type': 'application/json',
+    }
+    response = test_client.post('api/tasks/{task_id}/completions/'.format(task_id=task_id),
+                                data=json.dumps(label_data),
+                                headers=headers)
+    assert response.status_code == 201
+
+
+def action_label_test(test_client, case_config):
+    """
+        test
+        make sure completion result same as planned
+    """
+    label_data = case_config['label_data']
+
+    response = test_client.get('/api/projects/1/task_ids/')
+    data = json.loads(response.data.decode('utf-8'))
+    task_id = data[-1]
+
+    project = goc_project()
+    filename = os.path.join(project.config.get('output_dir', None),
+                            '{task_id}.json'.format(task_id=task_id))
+    with open(filename) as json_file:
+        completion = json.load(json_file)
+        assert completion.get('completions', {})[0].get('result', []) == label_data['result']
+
+
+def action_export(test_client, case_config):
+    """
+        export data
+        make sure it is in project directory
+    """
+    response = test_client.get('/api/export?format=JSON')
+    assert response.status_code == 200
+
+
+def action_export_test(test_client, case_config):
```

Comment:
seems that it's a duplicate or `action_export`

Reply:
currently yes,
but we've a plan to comare zip output,
skipped now due to mvp solution
problem is - result.json contains datetime stamps,
that obviously make zips different